### ISSUE TITLE
RFDB is difference with figure in the paper.

### ISSUE THREAD

**Issue by GuideWsp** [njulj/RFDN#3]
Finally, I found the code of the RFDB is not the same as the paper.

**Comment by njulj** [comment]
The only difference is that we did not use big residual connections in RFDB, which we found is slightly better in the challenge

**Comment by GuideWsp** [comment]
Yes, I had try it as the paper, finally I got a 30.47dB on DIV2K-val(YCbCr), the psnr of your model is 30.49dB, they are very similar. 
I just train it by L1, not finetune by L2.
Whatever, it is a very good job.

**Comment by scutlrr** [comment]
> Yes, I had try it as the paper, finally I got a 30.47dB on DIV2K-val(YCbCr), the psnr of your model is 30.49dB, they are very similar.
> I just train it by L1, not finetune by L2.
> Whatever, it is a very good job.

I also used the provided RFDN_AIM.pth file for testing, and the test result on DIV2K_val (only Y channel) was 30.517db. Maybe it is caused by the different calculation methods of psnr.

**Comment by hcleung3325** [comment]
> Yes, I had try it as the paper, finally I got a 30.47dB on DIV2K-val(YCbCr), the psnr of your model is 30.49dB, they are very similar.
> I just train it by L1, not finetune by L2.
> Whatever, it is a very good job.

May I know where can we get the training code?

**Comment by aiyunhug** [comment]
> 唯一的区别是，我们在RFDB中没有使用大的残差连接，我们发现这在挑战中略好一些。

why is 'self.remaining_channels = in_channels' in class RFDB？

**Comment by Liiiiaictx** [comment]
@scutlrr 大佬，这个RFDN怎么基于EDSR框架训练的，需要把这个仓库的文件拷贝到EDSR仓库吗？其次RFDN的环境配置是什么样的？

**Comment by Liiiiaictx** [comment]
@aiyunhug 大佬，这个RFDN怎么基于EDSR框架训练的，需要把这个仓库的文件拷贝到EDSR仓库吗？其次RFDN的环境配置是什么样的？
